runtime/src/snapshot_utils.rs

```diff
@@ -179,19 +220,104 @@ pub fn package_snapshot<P: AsRef<Path>, Q: AsRef<Path>>(
     archive_format: ArchiveFormat,
     snapshot_version: SnapshotVersion,
     hash_for_testing: Option<Hash>,
-) -> Result<AccountsPackagePre> {
-    // Hard link all the snapshots we need for this package
+) -> Result<AccountsPackagePre>
+where
+    P: AsRef<Path>,
+    Q: AsRef<Path>,
+{
+    info!(
+        "Package snapshot for bank: {} has {} account storage entries",
+        bank.slot(),
+        snapshot_storages.len()
+    );
+
     let snapshot_tmpdir = tempfile::Builder::new()
         .prefix(&format!("{}{}-", TMP_SNAPSHOT_PREFIX, bank.slot()))
         .tempdir_in(snapshot_path)?;
 
-    // Create a snapshot package
+    do_package_snapshot(
+        bank,
+        snapshot_files,
+        status_cache_slot_deltas,
+        snapshot_package_output_path,
+        snapshot_storages,
+        archive_format,
+        snapshot_version,
+        hash_for_testing,
+        snapshot_tmpdir,
+    )
+}
+
+/// Package up slot snapshot files, snapshot storages, and slot deltas for an incremental snapshot.
+#[allow(clippy::too_many_arguments)]
+fn package_incremental_snapshot<P, Q>(
+    bank: &Bank,
+    incremental_snapshot_base_slot: Slot,
+    snapshot_files: &SlotSnapshotPaths,
```

Comment:
`snapshot_files -> slot_snapshot_path` since it's only one slot?

Reply:
Done!